.github/actions/kind/action.yml

```diff
@@ -48,6 +48,32 @@ runs:
         kubectl cluster-info
         kubectl describe nodes
 
+    - name: Create Ingress resource
```

Comment:
I think it'd be more appropriate to have that Ingress created in the TestMNISTRayClusterSDK test itself.